service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/auth/AuthenticationType.java

```diff
@@ -6,12 +6,12 @@
 /**
  * Enum for the type of authentication used.
  * SAS - shared access signature
- * SELF_SIGNED - self signed certificate
+ * selfSigned - self signed certificate
  * CERTIFICATE_AUTHORITY - certificate authority signed certificate
  */
 public enum AuthenticationType
 {
     SAS,
-    SELF_SIGNED,
+    selfSigned,
```

Comment:
Do we have no control over how these enum values are serialized so we get the proper casing for the wire and not have to compromise on casing standards for Java enum values?

@timtay-microsoft

Reply:
I'd rather we tag the enum value ```SELF_SIGNED``` with 

```@SerializedName("selfSigned")``` so that we serialize/deserialize with "selfSigned" but don't cause a breaking change on the enum value itself.

Reply:
Good point; we can't introduce a breaking change.

Reply:
@yilmaznaslan do you want to resubmit with just that one line change? We do very much appreciate that you've taken the time to submit a PR, of course. :)

Reply:
I have built and tested it with `@SerializedName("selfSigned")`, it worked out :) I have submitted the change